load-test-framework/src/main/java/com/google/pubsub/clients/pubsubemulator/PubSubEmulatorPublisherTask.java

```diff
@@ -0,0 +1,59 @@
+package com.google.pubsub.clients.pubsubemulator;
+
+import com.beust.jcommander.JCommander;
+import com.google.api.gax.batching.BatchingSettings;
+import com.google.api.gax.core.NoCredentialsProvider;
+import com.google.api.gax.rpc.TransportChannelProvider;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.protobuf.util.Durations;
+import com.google.pubsub.clients.common.LoadTestRunner;
+import com.google.pubsub.clients.common.LoadTestTopicAdminSettings;
+import com.google.pubsub.clients.common.PublisherTask;
+import com.google.pubsub.flic.common.LoadtestProto.StartRequest;
+import com.google.pubsub.v1.TopicName;
+import java.io.IOException;
+import org.threeten.bp.Duration;
+
+
+class PubSubEmulatorPublisherTask extends PublisherTask {
```

Comment:
My point was to add an argument to how CpsPublisherTask was created so that it can just run against the emulator, so that there don't have to be changes to the number of tasks, flags, etc. i.e. just adding a parameter to the proto like message_size or topic.